AppOne/AppOne/View/HomeVC.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  HomeVC.swift
+//  AppOne
+//
+//  Created by Asapu, Ajaikumar on 19.11.22.
+//
+
+import UIKit
+
+final class HomeVC: UIViewController {
+    private lazy var brandsCollection: UICollectionView = {
+        let collectionV = UICollectionView(
+            frame: view.bounds,
```

Comment:
should be set to .zero to let Auto Layout do the sizing

Reply:
Ok